### PR TITLE
CRS-154 Allow access to our test environment from outside the VPN (do not restrict IP's)

### PR DESCRIPTION
**Pull request by yasinmustafa**

### CODE REVIEW

File: helm_deploy/values-dev.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ generic-service:
   ingress:
     host: calculate-release-dates-dev.hmpps.service.justice.gov.uk
     enable_allow_list: false
```
Comment (timharrison-moj) [inline helm_deploy/values-dev.yaml:9]: Can take this line out too now.

**Review by timharrison-moj — DISMISSED** [review]
LGTM. Can remove one line but approved - it won't harm anything.

File: helm_deploy/values-dev.yaml (hunk 1 above)
Comment (yasinmustafa) [reply helm_deploy/values-dev.yaml]: Thanks @timharrison-moj - will do

**Review by timharrison-moj — APPROVED** [review]
LGTM

**Pull request merged**